embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/spans/EmbraceSpanImplTest.kt

```diff
@@ -81,16 +86,31 @@ internal class EmbraceSpanImplTest {
     fun `validate span stopped state`() {
         with(embraceSpan) {
             assertTrue(start())
+            val expectedStartTimeMs = fakeClock.now()
+            fakeClock.tick(200)
             assertTrue(stop())
+            val expectedEndTimeMs = fakeClock.now()
+            assertSnapshot(
```

Comment:
Snapshotting a stopped span shouldn't really happen, but I didn't want to add more logic to prevent it, so I'm testing that it works and if some race condition makes this possible, no big deal - it'll be valid.